test/blackbox-tests/test-cases/directory-targets/install-dir-target.t

```diff
@@ -0,0 +1,27 @@
+Allow directories to be installable
+
+  $ cat >dune-project <<EOF
+  > (lang dune 3.0)
+  > (package (name foo))
+  > (using directory-targets 0.1)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (install
+  >  (dirs rules/bar)
```

Comment:
Just to clarify: is this supposed to install subdirectories of `rule/bar` too?

Reply:
Yes, that is definitely the right behavior.

Reply:
:+1: